Tasks/VsTest/testselectorinvoker.ts

```diff
@@ -7,7 +7,7 @@ import { Helper } from './helpers'
 let perf = require('performance-now');
 
 export class TestSelectorInvoker {
-    public publishCodeChanges(tiaConfig: models.TiaConfiguration, testCaseFilterFile: string, taskInstanceIdentifier: string): number {
+    public publishCodeChanges(tiaConfig: models.TiaConfiguration, vstestConfig: models.VsTestConfigurations, dtaTestConfig: models.DtaTestConfigurations, testCaseFilterFile: string, taskInstanceIdentifier: string): number {
```

Comment:
>dtaTestConfig [](start = 109, length = 13)

you should pass the proxyInterface  or vstestConfig (dtaconfig inherits vstestconfig right ?)

Reply:
Both dtaconfig and vstestconfig extend testconfig. dtaconfig does not inherit vstestconfig